src/vorta/views/repo_add_dialog.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, parent=None):
         self.saveButton = self.buttonBox.button(QDialogButtonBox.StandardButton.Ok)
         self.saveButton.setText(self.tr("Add"))
 
-        self.buttonBox.rejected.connect(self.close)
+        self.buttonBox.rejected.connect(lambda: (QApplication.instance().jobs_manager.cancel_all_jobs(), self.close()))
```

Comment:
This cancels all jobs, not just the borg info.

Reply:
Hi! As you said this will cancel all the jobs, I've made some changes, how can I test those exactly ? the scheduled jobs keep running, did you mean the backup ? like when we adding a repo and cancel it by the cancel button, if there's any backup job running for the other repo it will get cancelled  too ?

Reply:
> like when we adding a repo and cancel it by the cancel button, if there's any backup job running for the other repo it will get cancelled too ?

Yes, exactly.

Reply:
> > like when we adding a repo and cancel it by the cancel button, if there's any backup job running for the other repo it will get cancelled too ?
> 
> Yes, exactly.

Hm, actually the change will not add a big repo if cancel button is triggered and it will also not affect the backup running in background, but for the small repo it still adds the repo if you wait and then click the cancel button but it doesn't the background backup job (I'll push the new change once trying another solution 😅 )

also how can I run migrations 🤔